lib/runners/VapidServer/index.js

```diff
@@ -80,6 +80,9 @@ class VapidServer extends Vapid {
       const ext = path.extname(ctx.path);
 
       switch (ext) {
+        case '':
+          ctx.type = 'text/html';
+          break;
```

Comment:
@JohJakob What's the thought here, with an empty case? Are there times where the extension is blank?

Reply:
@srobbin Yes, this accounts for all index.html files that don’t get served with their full path.